samples/winapi_filefind/main.c

```diff
@@ -38,24 +39,29 @@ int main(void)
         } else {
             debugPrint("File     : ");
         }
-
         debugPrint("%s\n", findFileData.cFileName);
     } while (FindNextFile(hFind, &findFileData) != 0);
 
     debugPrint("\n");
 
-    DWORD error = GetLastError();
-    if (error == ERROR_NO_MORE_FILES) {
+    DWORD noMoreFilesError = GetLastError();
+    if (noMoreFilesError == ERROR_NO_MORE_FILES) {
```

Comment:
This variable name is bad, because it's not always `ERROR_NO_MORE_FILES`, which the name implies.

Instead, `findFileError` is a better name, because it reflects that it's sort-of synchronized with `findFileData`

Reply:
Using `findFileError` for this